src/HttpServer.cpp

```diff
@@ -152,7 +152,8 @@ HttpRequestHandler HttpWorker::_chooseHandler(HttpRequest& req) {
             int mvector[32];
             const size_t mvectorItems = sizeof(mvector) / sizeof(mvector[0]);
             int rc = pcre_exec(listItem->regexp, nullptr, query, query_len, 0, 0, mvector, mvectorItems);
-            if(rc >= 0) { /* match! */
+            if(rc >= 0) {
+                /* match! */
```

Comment:
Are you sure this is a right formatting after running `make format` ? I have Clang 4.0.0 and it doesn't change this. What version of Clang are you using?